src/test/java/uk/gov/ons/ctp/response/action/endpoint/ActionEndpointIT.java

```diff
@@ -279,7 +353,7 @@ private SampleAttributesDTO createSampleAttributesMock(UUID sampleUnitId) throws
     log.debug("sample_attributes to mock = " + sample_attributes_dto);
 
     this.wireMockRule.stubFor(
-        get(urlPathMatching(String.format("/samples/%s/attributes", sampleUnitId)))
+        get(urlPathMatching("/samples/(.*)/attributes"))
```

Comment:
I can't see a reason for this change?

Reply:
@AdamHawtin If you rerun the test and you have messages in your queue, WireMock ends up matching against the first message in that queue, rather than the message the test is putting into the queue. This causes test failures.